features/webapp-authenticator-framework/org.wso2.carbon.webapp.authenticator.framework.server.feature/src/main/resources/dbscripts/cdm/h2.sql

```diff
@@ -62,14 +62,14 @@ CREATE TABLE IF NOT EXISTS DM_POLICY_OPERATION (
     DM_OPERATION (ID) ON DELETE NO ACTION ON UPDATE NO ACTION
 );
 
-CREATE TABLE IF NOT EXISTS DM_DEVICE_OPERATION_MAPPING (
+CREATE TABLE IF NOT EXISTS DM_ENROLMENT_OPERATION_MAPPING (
     ID INTEGER AUTO_INCREMENT NOT NULL,
-    DEVICE_ID INTEGER NOT NULL,
+    ENROLMENT_ID INTEGER NOT NULL,
     OPERATION_ID INTEGER NOT NULL,
     STATUS VARCHAR(50) NULL,
     PRIMARY KEY (ID),
-    CONSTRAINT fk_dm_device_operation_mapping_device FOREIGN KEY (DEVICE_ID) REFERENCES
-    DM_DEVICE (ID) ON DELETE NO ACTION ON UPDATE NO ACTION,
+    CONSTRAINT fk_dm_device_operation_mapping_device FOREIGN KEY (ENROLMENT_ID) REFERENCES
+    DM_ENROLMENT (ID) ON DELETE NO ACTION ON UPDATE NO ACTION,
```

Comment:
Why ON DELETE NO ACTION? shouldn't this be ON DELETE CASCADE?
Why ON UPDATE NO ACTION? shouldn't this be ON UPDATE CASCADE?